### DISCUSSION TITLE
Account suspended<title>

### DISCUSSION THREAD

**Discussion by GuilhermeL12** [community/community#68752] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Galera tudo bem?

Tive minha conta suspensa o nome dela é @melhergui12 não sei o que aconteceu, simplismente foi banida. Alguém pode me ajudar, eu tinha muitos repositorios nessa conta 

![image](https://github.com/community/community/assets/132394907/856fd959-d369-4029-bdba-95162aa5c219)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @GuilhermeL12, 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). The community is unable to assist with nor escalate your ticket, but we can assure you once open it will be in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
